examples/website/src/shared/navigation/navigationBar.tsx

```diff
@@ -261,7 +261,9 @@ export function NavigationBar({itemsCategory, itemsIntern}: NavigationBarProps)
                 <div key={index}>
                   <Link
                     className={css(LinkStyles)}
+                    target={'_blank'}
```

Comment:
`target="_blank"` only when href.

Reply:
I made a distinction now between an internal link which has `target="_self"` and an external link which has `target="_blank". Can I leave it with that or should I really only use `target` when we're dealing with an external link?

Reply:
👍🏾  `target="self"` is bueno